<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_lancamentos</class>
 <widget class="QMainWindow" name="frm_lancamentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lançamentos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edt_doc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_desc1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_desc2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_valor">
    <property name="geometry">
     <rect>
      <x>512</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doc Nº</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inserir">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>54</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_tipo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SAIDA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ENTRADA</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>711</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_excluir">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salvar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edt_ID">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2021</year>
      <month>12</month>
      <day>7</day>
     </datetime>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::MonthSection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
    <property name="date">
     <date>
      <year>2021</year>
      <month>12</month>
      <day>7</day>
     </date>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>São Paulo-SP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gravataí-RS</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
